app/furniture/marketplace/checkout.rb

```diff
@@ -16,9 +16,6 @@ def create_stripe_session(success_url:, cancel_url:)
         mode: "payment",
         success_url: success_url,
         cancel_url: cancel_url,
-        shipping_address_collection: {
-          allowed_countries: ["US"],
-        },
```

Comment:
We don't want to get it from Stripe anymore, since we get it before hand.

Reply:
I assume: "get it before hand." from the user via this code change that allows the user to input their delivery address in a field.

Reply:
Yep